code/modules/mob/living/carbon/human/species/shadow.dm

```diff
@@ -16,7 +17,7 @@
 		"brain" = /obj/item/organ/brain
 		)
 
-	flags = NO_BLOOD | NO_BREATHE | NO_SCAN
+	flags = NO_BLOOD | NO_BREATHE | NO_SCAN | NO_POISON
```

Comment:
Is this the species that shadowlings are? They definitely don't need poison immunity.